### DISCUSSION TITLE
Can't adjust the height of tab labels with Tabs component?

### DISCUSSION THREAD

**Discussion by AleMCortez** [TiendaNube/nimbus-design-system#249] [Q&A]
I'm currently using the Tabs component and created this component:

<img width="579" alt="image" src="https://github.com/user-attachments/assets/64658d08-3a61-4ac5-8cdb-883d18985512">

If I click in one of the Tab titles, it looks like this:

<img width="579" alt="image" src="https://github.com/user-attachments/assets/59716a72-8adb-4584-aebc-8e3f4d1ceafb">

This is because I manually added a blank space above the Tab labels with a Box component because we want that section to be a bit taller, but this causes it to look wrong when the tab label component is selected because the selection doesn't go all the way up. A solution would be to make the actual label component taller, but for what I see, neither the Tabs component nor the Tabs.Item component supports something like this.

As a side note, if I go to Nimbus documentation and try to see a component's props, it always looks something like this:

<img width="900" alt="image" src="https://github.com/user-attachments/assets/83d81bf8-46dc-4c97-910d-08e69a1eea1a">

**Comment by harrytiendanube** [comment]
@AleMCortez Could this be because the tabs are inside a card, and the space above them is due to the spacing the card adds?
Do you have any code examples?
